bin/function_override_wrapper.sh.erb

```diff
@@ -1,6 +1,9 @@
 /usr/bin/env bash -c '
 # load in command and function overrides
-for override_file in <%=function_override_path_binding_for_template%>; do
+system_ls=$(which ls)
+system_grep=$(which grep)
```

Comment:
Oh yeah, I remember that now. It's been too long. That's why I had
 ```
pattern="(/.*)( |$).*"
 +[[ $(whereis ls) =~ $pattern ]]
 +system_ls=${BASH_REMATCH[1]}
```
originally. It worked on OSX and Ubuntu I thought, but maybe not all versions of ubuntu